test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/ContractBurnHTSSuite.java

```diff
@@ -359,10 +408,26 @@ private HapiApiSpec HSCS_PREC_020_rollback_burn_that_fails_after_a_precompile_tr
 													.hasKnownStatus(CONTRACT_REVERT_EXECUTED));
 								}),
 
-						childRecordsCheck("contractCallTxn", CONTRACT_REVERT_EXECUTED, recordWith()
-										.status(REVERTED_SUCCESS),
+						childRecordsCheck("contractCallTxn", CONTRACT_REVERT_EXECUTED,
+								recordWith()
+										.status(REVERTED_SUCCESS)
+										.contractCallResult(
+												resultWith()
+														.contractCallResult(htsPrecompileResult()
+																.forFunction(HTSPrecompileResult.FunctionType.BURN)
+																.withStatus(SUCCESS)
+																.withTotalSupply(1)
+														)
+										),
 								recordWith()
 										.status(INSUFFICIENT_SENDER_ACCOUNT_BALANCE_FOR_CUSTOM_FEE)
+										.contractCallResult(
+												resultWith()
+														.contractCallResult(htsPrecompileResult()
+//																.forFunction(HTSPrecompileResult.FunctionType.FAILED)
```

Comment:
Probably we don't need this comment?

Reply:
Yes, I wanted to ask about the commented row : is "HTSPrecompileResult.FunctionType.FAILED" with adequate value?